praw/objects.py

```diff
@@ -636,6 +636,33 @@ class Message(Inboxable):
 
     """A class for private messages."""
 
+    @staticmethod
+    @restrict_access(scope='privatemessages')
+    def from_id(reddit_session, message_id, *args, **kwargs):
+        """Request the url for a Message and return a Message object.
+
+        :param reddit_session: The session to make the request with.
+        :param message_id: The ID of the message to request.
+
+        The additional parameters are passed directly into
+        :meth:`.request_json`.
+
+        """
+        # Reduce fullname to ID if necessary
+        message_id = message_id.replace('t4_', '')
```

Comment:
No need to check. You can simply use:

`message_id = message_id.split('_', 1)[-1]`

The separator will always be an underscore, unless reddit makes a significant change. The prefixes, however, are assigned during DB creation, which for many local installs results in different prefix numbers for the objects.